static/js/tools/choropleth/choropleth.tsx

```diff
@@ -173,6 +181,13 @@ class ChoroplethMap extends Component {
         return "gray";
       }
     });
+
+    // Update title.
```

Comment:
Add a TODO to use a React component / function to do this instead of setting innerHTML directly

Reply:
Added.